File: five-little-monkeys.js
var numberOfMonkeys = ["Five", "Four", "Three", "Two", "One"];

numberOfMonkeys.forEach(function(value) {
    if(value.indexOf("One")) {
        console.log(value + " little monkey jumping on the bed");
        console.log("They fell off and bumped their head");
    } else {
        console.log(value + " little monkeys jumping on the bed");
        console.log("One fell off and bumped their head");
    }
    
    console.log("Mama called the doctor")
    console.log("And the doctor said")
    
    if(value.indexOf("One")) {
        console.log("Put those monkeys right to bed\n")
    } else {
        console.log("No more monkeys jumping on the bed\n")
    }
});

File: five-little-monkeys.rs
fn main() {
	let number_of_monkeys = vec![ "One", "Two", "Three", "Four", "Five" ];
	
	for monkey in number_of_monkeys.iter().rev() {
		if monkey.to_string() == "One" {
			println!("{} little monkey jumping on the bed", monkey);
			println!("They fell off and bumped their head");
		} else {
			println!("{} little monkeys jumping on the bed", monkey);
			println!("One fell off and bumped their head");
		}
		
		println!("Mama called the doctor,");
		println!("And the doctor said");
		
		if monkey.to_string() == "One" {
			println!("Put those monkeys right to bed\n");
		} else {
			println!("No more monkeys jumping on the bed\n");
		}
	}
}